content/millennium/r4/bulk-data/group-export.md

```diff
@@ -0,0 +1,305 @@
+
+## Kick-off Request
+
+Kicks off a bulk export request for a group of patients:
+
+    GET /Group/<group_id>/$export
+
+Operation is defined at:
+
+    GET /OperationDefinition/group-export?_format=json
+
+_Implementation Notes_
+
+* Bulk data export requests do not currently honor the Accept-Language header; 'en-US' will be used.
+* Only one bulk export request for a client application and tenant combination may be in-progress at a single time.
+
+### Authorization Types
+
+<%= authorization_types(provider: false, patient: false, system: true) %>
+
+_Notes_
+
+* Token does NOT require the `Group.read` scope
+* Token must contain system read scopes for all resources requested in the `_type` parameter
+
+### Parameters
+
+Name                    | Required? | Type        | Description
+------------------------|-----------|-------------|---------------------------------------------------------------------------------------------------
+`_type`                 | No        | [`string`]  | A string of comma-delimited FHIR resource types. Example: `_type=Patient,Encounter,Location`
+`_typeFilter`           | No        | [`string`]  | A string of comma-delimited FHIR Rest queries provided in combination with `_type`. Example: `_typeFilter=Condition?clinical-status=active,Encounter?status=planned`
+`_since`                | No        | [`instant`] | Resources updated after this time will be included in the response. Example: `2021-03-18T19:19:42.000Z`
+`includeAssociatedData` | No        | [`string`]  | When provided, the group export will return or omit a pre-defined set of FHIR resources associated with the request. A string of comma-delimited values. Example: `includeAssociatedData=LatestProvenanceResources`
+`_outputFormat`         | No        | [`string`]  | The format for the requested bulk data files to be generated. Example: `_outputFormat=application/ndjson`
+
+_Notes_
+
+* `_type` valid parameters are `AllergyIntolerance`, `CarePlan`, `CareTeam`, `Condition`, `Device`, `DiagnosticReport`, `DocumentReference`, `Encounter`, `Goal`, `Immunization`, `Location`, `MedicationRequest`, `Patient`, `Procedure`, `Observation`, `Organization`, `Practitioner`, `Provenance`
+* When the `_type` parameter contains a reference resource (`Location`, `Organization`, `Practitioner`) or associated resource (`Provenance`), at least one of the following resources must also be provided: `AllergyIntolerance`, `CarePlan`, `CareTeam`, `Condition`, `Device`, `DiagnosticReport`, `DocumentReference`, `Encounter`, `Goal`, `Immunization`, `MedicationRequest`, `Observation`, `Patient`, `Procedure`
+* When the `_type` parameter is not provided, the resources to be exported will be determined by the scopes provided in the authorization token.
+* When the `_typeFilter` parameter is provided, the `_type` parameter MUST be provided as well.
+* `_typeFilter` does not support queries for the following resources: `Location`, `Organization`, `Patient`, `Provenance`
```

Comment:
how about Practitioner? It is a reference resource  and retrieved by id so filtering cannot be applied